templates/zerver/integrations/mercurial.md

```diff
@@ -0,0 +1,74 @@
+Get Zulip notifications when you `hg push`!
+
+First, create the stream you’d like to use for Mercurial notifications, and subscribe all interested parties to this
+stream. We recommend the name `commits`.
+
+Next, on your {{ settings_html|safe }}, create a Mercurial bot.
+
+Then:
+
+1.  Download and install our [Python bindings](/api) on the server where the master Mercurial repository lives.
+
+2.  Edit the `hg/.hgrc` configuration file for this master Mercurial repository and add the following sections,
+using the credentials for your Mercurial bot and setting the appropriate path to the integration hook if it installs
+in a different location on this system:
+
+        [hooks]
+        changegroup = python:/usr/local/share/zulip/integrations/hg/zulip-changegroup.py:hook
+
+        [zulip]
+        email = "hg-bot@example.com"
+        api_key = "0123456789abcdefg"
+        stream = "commits"
+        {% if api_site_required %}site = {{ external_api_uri_subdomain }}{% endif %}
+
+That’s all it takes for the basic setup! On the next `hg push`, you’ll get a Zulip update for the changeset.
+
+### More configuration options
+
+The Mercurial integration also supports:
+
+-   linking to changelog and revision URLs for your repository’s web UI
+-   branch whitelists and blacklists
+
+#### Web repository links
+
+If you’ve set up your repository to be [browsable via the
+web](http://mercurial.selenic.com/wiki/QuickStart#Network_support), add a `web_url` configuration option to the
+`zulip` section of your master `.hg/hgrc` to get changelog and revision links in your Zulip notifications:
+
+    [zulip]
+    email = "hg-bot@example.com"
+    api_key = "0123456789abcdefg"
+    stream = "commits"
+    web_url = "http://hg.example.com:8000/"
+    {% if api_site_required %}site = {{ external_api_uri_subdomain }}{% endif %}
+
```

Comment:
It renders it as code block if indented. Indentation was needed to keep in line with previous version of doc.